.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "utils/cloudio/storage-manager"]
+	path = utils/cloudio/storage-manager
+	url = git@github.com:pleblanc1976/storage-manager
```

Comment:
Can we please:

1. Move this to a more official repo (I can get one created if needed)
2. Add appropriate license headers to the source files in that repo (I can help with BSL headers if needed)

Reply:
Yes both of those need to be done.  Should we do that in this PR, or add a ticket + PR for follow-on mods?